pandas/core/computation/pytables.py

```diff
@@ -556,7 +547,7 @@ def __init__(
                 else:
                     w = _validate_where(w)
                     where[idx] = w
-            _where = " & ".join(map("({})".format, com.flatten(where)))
+            _where = " & ".join([f"({w})" for w in com.flatten(where)])
```

Comment:
Can you use a generator expression here instead of a list? Would be more equivalent to the original